modules/videopress/shortcode.php

```diff
@@ -36,6 +38,33 @@ public static function initialize() {
 		return self::$instance;
 	}
 
+	/**
+	 * Filters old <embed> codes out of the content
+	 *
+	 * @param string $content the content.
+	 * @return string filtered content
+	 */
+	public function reverse_old_embeds( $content ) {
+		$regex   = '%<embed[^>]*+>(?:\s*</embed>)?%i';
+		$content = preg_replace_callback(
+			$regex,
+			function( $matches, $orig_html = null ) {
+				$embed_code  = $matches[0];
+				$url_matches = array();
+
+				// get video ID from flash URL.
+				$url_matched = preg_match( '/src="http:\/\/v.wordpress.com\/([^"]+)"/', $embed_code, $url_matches );
```

Comment:
Do you know if the old embed codes also supported the additional query strings supported by the newer embed, e.g. `at=9&loop=1&autoPlay=1`? We could catch those as well, if present?

Reply:
I've dug around the old VideoPress site and videop2. I don't see anything suggesting it did yet.